app/src/main/java/com/google/android/samples/socialite/ui/player/VideoPlayerScreen.kt

```diff
@@ -179,7 +180,17 @@ private fun VideoPlayer(
                 // set source rect hint, aspect ratio and remote actions
                 val sourceRect = layoutCoordinates.boundsInWindow().toAndroidRectF().toRect()
                 builder.setSourceRectHint(sourceRect)
-                builder.setAspectRatio(Rational(sourceRect.width(), sourceRect.height()))
+                var aspectRatio = Rational(sourceRect.width(), sourceRect.height())
+                val upperAspectRatioBound = Rational(2.39.toInt(), 1)
+                val lowerAspectRatioBound = Rational(1, 2.39.toInt())
```

Comment:
2.39.toInt() will always be 2. We can say Rational(239, 100) and Rational(100, 239) for exact values.

Reply:
makes sense!!